### PR TITLE
Adds input compatibility with 4-dim uint8 torch Tensor; absolute -> r…

### PR DESCRIPTION
**Pull request by voletiv**
…elative imports

### CODE REVIEW

**Comment by toshas** [comment]
Hey Vikram, thank you for your interest to the package! I will get back to your pull request after June 10. However, before you spend more time on the changes in this PR, I wanted to let you know that it might have been better to start a discussion via "GitHub Issues", and proceed with isolated PRs after everyone is on the same page with the submitted issues and about where this project goes. I'm sorry that I did not identify this via the "CONTRIBUTING" file, which apparently is necessary to prevent issues with collaboration - I will do so later. 

From what I see now, adding tensors input support creates more issues than solves, as there is shuffling and other dataloader logic to the inputs, besides it can be solved in one line using the `torch.utils.data.TensorDataset` on the user side. As to the other aspects, it is much harder to get a PR merged when it is a "package deal", hence again the "Github Issues" argument.

**Comment by voletiv** [comment]
Hi Anton, sure I'll close this PR for now.

I've made many changes after the one I submitted at the time, which I'm currently using for my purposes. Thank you so much for this repo btw, it's great to easily use code to get to SOTA calculations for IS, FID, KID in PyTorch.

I'll launch individual issues for the different items I've changed and you can accept whichever you think is correct.

I've tried using TensorDataset, but the problem is that it returns a tuple of the Tensor, which goes against the repo's principle of the dataset returning just the tensor. That's why I've had to include the option of a uint8 4D Tensor as input. I'm not sure what "shuffling and other dataloader logic to the inputs" can't be applied to a DataLoader of a Tensor, as far as I know it should be compatible with all of them.
